packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.types.ts

```diff
@@ -444,6 +444,12 @@ export interface IContextualMenuItem {
    * Optional prop to make an item readonly which is disabled but visitable by keyboard, will apply aria-readonly and some styling. Not supported by all components
    */
   inactive?: boolean;
+
+  /**
+   * Text description for the menu item to display
+   * @deprecated Use 'text' instead.
```

Comment:
Use backticks instead of singlequote around text (you can use markdown in JSDoc)